pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -295,27 +295,46 @@ def _check_config_and_set_final_flags(
             self._precision_flag = precision
 
         if plugins:
+            plugins_flags_count = {
+                "strategy": 0,
+                "precision_plugin": 0,
+                "precision": 0,
+                "checkpoint_io": 0,
+                "cluster_env": 0,
+            }
             for plugin in plugins:
                 if isinstance(plugin, Strategy) or isinstance(plugin, str) and plugin in self._registered_strategies:
                     self._strategy_flag = plugin
                     rank_zero_deprecation(
                         f"Passing {plugin} `strategy` to the `plugins` flag in Trainer has been deprecated"
                         f" in v1.5 and will be removed in v1.7. Use `Trainer(strategy={plugin})` instead."
                     )
+                    plugins_flags_count["strategy"] += 1
```

Comment:
Not a huge fan of this approach.

You could also do

```python
if self._strategy_flag is not None:
	raise MisconfigurationException("Received multiple values for strategy in `plugins` ...")

Reply:
Not really.. the self._strategy_flag could be set by strategy flag at L242-L276
The duplication of plugin[strategy] and strategy flag has been handled in L242-L276 as well

Reply:
Same for precision_plugin_flag, it could be set above already. We will need a seperate `count` variable to track whether plugin has multiple same type. 
Any suggestion @kaushikb11 :)?

Reply:
I like this way because we can tell user all the duplications at one go